changelog/2702.breaking.rst

```diff
@@ -0,0 +1,3 @@
+Added a `__str__` method to the ``CustomParticle``
```

Comment:
```suggestion
Added a ``__str__`` method to the |CustomParticle|
```

Reply:
Thank you! I was struggling to understand that Sphynx error.